.gitmodules

```diff
@@ -1,8 +1,3 @@
-[submodule "llm/llama.cpp/ggml"]
```

Comment:
Removing this is premature. This is the first mention of deprecating ggml and it removes support completely. There's no path forward for individuals who use ggml-only models other than not upgrading. Instead, it should warn first and only after a couple release should it remove support